src/bookmarks/bookmarks.controller.ts

```diff
@@ -16,6 +16,9 @@ import { JwtAuthGuard } from 'src/auth/guard/jwt-auth.guard';
 import RequestWithUser from 'src/auth/interface/request-with-user.interface';
 import { Pagination } from 'src/interfaces/pagination.interface';
 import { PaginationDto } from 'src/dto/pagination.dto';
+import { Roles } from 'src/enums/roles.enum';
```

Comment:
could you please configure absolute paths so that they skip `src` folder
meaning imports should be:
```suggestion
import { Roles } from 'enums/roles.enum';
```